internal/cmd/integration-tests/utils.go

```diff
@@ -56,7 +56,7 @@ func buildAgent() {
 }
 
 func setupEnvironment() {
-	success := executeCommand("docker-compose", []string{"up", "--abort-on-container-exit"}, "Setting up environment with Docker Compose")
+	success := executeCommand("docker", []string{"compose", "up", "--abort-on-container-exit"}, "Setting up environment with Docker Compose")
```

Comment:
@ptodev I think this is missing `-d` to detach the process so it continue running.